docs/changelogs/latest.rst

```diff
@@ -15,34 +15,72 @@ SIMP Community Edition (CE) 6.5.0-Beta
 
   PageBreak
 
+
+OS compatibility
+----------------
+
+.. contents::
+  :depth: 2
+  :local:
+
 This release is known to work with:
 
   * CentOS 6.10 x86_64
   * CentOS 7.0 2003 x86_64
-  * CentOS 8.2 2004 x86_64 (client systems only)
+  * CentOS 8.2 2004 x86_64 — :ref:`client systems only<changelog-6-5-0-el8-client-only>`
   * OEL 6.10 x86_64
   * OEL 7.8 x86_64
-  * OEL 8.2 x86_64 (client systems only)
+  * OEL 8.2 x86_64 — :ref:`client systems only<changelog-6-5-0-el8-client-only>`
   * RHEL 6.10 x86_64
   * RHEL 7.8 x86_64
-  * RHEL 8.2 x86_64 (client systems only)
+  * RHEL 8.2 x86_64 — :ref:`client systems only<changelog-6-5-0-el8-client-only>`
+
+
+Important OS compatibility limitations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 OS compatibility is subject to the following limitations:
 
-* EL8 support is currently limited to Puppet agents
 
-  * This release does **not** support managing an EL8 SIMP Server or installing
-    SIMP from an EL8 ISO.
-  * EL8 management is supported by all Puppet modules provided as core
-    dependencies of the :package:`simp` RPM.
-  * Not all modules provided by the :package:`simp-extras` RPM have been updated
-    for EL8.
-  * EL8 updates to the remaining :package:`simp-extras` modules will be phased
-    in over future SIMP releases.
-  * Support for managing an EL8 SIMP/Puppet server and installing from
-    EL8 ISOs will be provided in a later SIMP release (SIMP 6.6.0).
 
-* Support for managing EL6 system is drawing down.
+.. _changelog-6-5-0-el8-client-only:
+
+EL8 support is CLIENT ONLY
+""""""""""""""""""""""""""
+
+This release introduces client-only EL8 support in the core Puppet modules.
+
+* EL8 support is limited to managing EL8 Puppet *agents*
+* with the core Puppet modules.
```

Comment:
@op-ct 
did you want a bullet between the agent and the with?
* EL8 support is limited to managing EL8 Puppet agents
* with the core Puppet modules.